scripts/security/scan.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+
+from github import Github
+from git import Git
```

Comment:
These dependencies have to be added to `requirements.txt`. Maybe an exclusive file for security. `requirements-security.txt`